middleware.js

```diff
@@ -298,6 +298,21 @@ async function requireNoOptOut(req, res, next) {
   }
 }
 
+async function requireMozAdmin(req, res, next) {
+  const user = req.user;
+
+  if (!user?.primary_email.includes("@mozilla.com")) {
+    console.error("non mozilla email");
+    return res.status(401).json({
+      error:
+        "You must be signed in with a mozilla.com email address to access this page",
+    });
+  } else {
+    next();
+    return;
+  }
```

Comment:
This doesn't need an `else` block, because if the code flowed into the above `if` block, it will already call `return` and never execute the code in `else`. Somewhat simpler form:

```suggestion
  if (!user?.primary_email.includes("@mozilla.com")) {
    console.error("non mozilla email");
    return res.status(401).json({
      error:
        "You must be signed in with a mozilla.com email address to access this page",
    });
  }
  next();
  return;
```

Reply:
👍🏽